app/models/answer.rb

```diff
@@ -32,4 +35,23 @@ def accept
   def sort_value
     accepted ? Float::INFINITY : votes_count
   end
+
+  def create_action_verb
+    "Answered a Question"
+  end
+
+  def update_action_verb
+    "Updated his/her Answer on Question"
```

Comment:
`his/her` might sound bizarre when reading it on someone profile. Imagine there are multiple such activity displayed together.

Maybe consider removing the personal pronoun.